app/frontend/localStorage.ts

```diff
@@ -1,6 +1,6 @@
 const localStorageVars = {
   WELCOME: 'dontShowDisclaimer',
-  INFO_MODAL: 'dontShowInfoModal_10.08.2023',
+  INFO_MODAL: 'dontShowInfoModal_01.10.2024',
```

Comment:
nitpick, we usually use here the european format
```suggestion
  INFO_MODAL: 'dontShowInfoModal_10.01.2024',
```